Editors/Audio/Presentation/AudioEditor/AudioEditorSettings.cs

```diff
@@ -0,0 +1,295 @@
+﻿using System.Collections.Generic;
+
+namespace Editors.Audio.Presentation.AudioEditor
+{
+    public class AudioEditorSettings
```

Comment:
This class should probably be a static? 
Also, the attributes should only have a get, not a set?